subcommands/targets/offline-update.go

```diff
@@ -84,6 +87,15 @@ func doOfflineUpdate(cmd *cobra.Command, args []string) {
 	fmt.Println("Successfully refreshed and downloaded TUF metadata")
 
 	if !ouTufOnly {
+		if !isDstDirClean(dstDir) {
+			fmt.Println("Destination directory already has update data")
+			if !ouAllowMultipleTargets {
+				subcommands.DieNotNil(errors.New(
+					"Please provide a clean destination directory or re-run with --allow-multiple-targets to add a new target to a directory which already has update data.\n" +
+						"Notice that multiple targets in the same directory is only supported in LmP >= v92."))
```

Comment:
Just IMHO this message is only needed for the `if !ouAllowMultipleTargets` code path.
I'm not strong on this, however, the error message seems cleaner in that case.
In contrast, when the command is run with `--allow-multiple-targets` , it might look confusing.

```suggestion
			if !ouAllowMultipleTargets {
				subcommands.DieNotNil(errors.New(`Destination directory already has update data.
Provide a clean destination directory or re-run with --allow-multiple-targets to add a new target to a directory which already has update data.
Notice that multiple targets in the same directory is only supported in LmP >= v92.`))
```

Reply:
@vkhoroz , I've added your suggestions. Error now looks like this:
```
ERROR: Destination directory already has update data.
Provide a clean destination directory or re-run with --allow-multiple-targets to add a new target to a directory which already has update data.
Notice that multiple targets in the same directory is only supported in LmP >= v92.
```